app/controllers/spotlight/about_pages_controller.rb

```diff
@@ -5,6 +5,10 @@ class AboutPagesController < PagesController
     load_and_authorize_resource through: :exhibit, instance_name: 'page'
     before_action :attach_breadcrumbs, except: [:update_contacts]
 
+    def clone
+      super
```

Comment:
I don't understand why we have to do this, but otherwise we get an undefined action error (or something along those lines).  Would love to get an extra pair of eyes on this.